shell/platform/darwin/ios/framework/Source/FlutterStandardCodec.mm

```diff
@@ -36,34 +55,53 @@ - (id)decode:(NSData*)message {
 
 #pragma mark - Codec for method channel
 
-@implementation FlutterStandardMethodCodec
+@implementation FlutterStandardMethodCodec {
+  FlutterStandardReaderWriter* _readerWriter;
+}
 + (instancetype)sharedInstance {
   static id _sharedInstance = nil;
   if (!_sharedInstance) {
-    _sharedInstance = [FlutterStandardMethodCodec new];
+    _sharedInstance =
```

Comment:
The shared instance will be garbage after the ARP activation since `-[FlutterStandardMethodCodec 
 withReaderWriter:]` returns an autoreleased object.